File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.py
def solution(babbling):
    check = ['aya', 'ye', 'woo', 'ma']
    answer = 0
    
    for v in babbling:
        for c in check:
            if c * 2 not in v:
                v = v.replace(c, ' ')
        
        if not v.split():
            answer += 1
        
    return answer

File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.java
class Solution {
    
    public int solution(String[] babbling) {
        String[] check = {"aya", "ye", "woo", "ma"};
        int answer = 0;

        for (String v : babbling) {
            String temp = v;

            for (String c : check) {
                if (!temp.contains(c + c)) {
                    temp = temp.replace(c, " ");
                }
            }

            if (temp.trim().isEmpty()) {
                answer++;
            }
        }

        return answer;
    }
}